map/separate_chaining_hashtable.py

```diff
@@ -0,0 +1,147 @@
+import unittest
+
+
+class Node(object):
+    key = None
+    value = None
+    next = None
```

Comment:
I think these attributes shouldn't be `class` attributes of `Node`. They should only be defined in `__init__`. Otherwise, only one copy gets initialized... or I maybe wrong..

Reply:
yeah, I changed this.